cfpb/cfgov-refresh
# New homepage: prevent tabbing to links in hidden carousel slides

When tabbing through the home page, focus is temporarily lost after the active slide's link because links in the inactive slides appear in the tab order. This ensures that focus goes directly from the active slide's link to the first of the slideshow thumbnail buttons. (GHE 63)

## Additions

- Hidden class for carousel items

## Changes

- Include visibility as well as opacity in the slide transitions

## Testing

1. Check out branch
2. Navigate to [home page](http://localhost:8000/?nhp=True)
3. Tab through the page and verify that focus goes directly from link in active tab to first of 4 carousel slide buttons
4. Click through carousel and verify that slide transitions work as expected



## Checklist


## Testing checklist

### Browsers


### Accessibility
